gobblin-hive-registration/src/main/java/org/apache/gobblin/hive/writer/MetadataWriterKeys.java

```diff
@@ -31,6 +31,7 @@ public class MetadataWriterKeys {
   public static final String GMCE_TOPIC_NAME = "gmceTopicName";
   public static final String GMCE_TOPIC_PARTITION = "gmceTopicPartition";
   public static final String GMCE_HIGH_WATERMARK = "gmceHighWatermark";
+  public static final String TABLE_COMPLETION_WATERMARK = "tableCompletionWatermark";
```

Comment:
Let's maybe just use `IcebergMetadataWriterConfigKeys.COMPLETION_WATERMARK_KEY` instead of defining a new key.